src/Workspaces/Core/Portable/FindSymbols/FindReferences/DependentTypeFinder.cs

```diff
@@ -177,7 +178,7 @@ internal static class DependentTypeFinder
                     type,
                     solution,
                     null,
-                    (t1, t2) => t1.BaseType == t2,
+                    (candidate, baseType) => AbstractReferenceFinder<INamedTypeSymbol>.OriginalSymbolsMatch(candidate.BaseType, baseType, solution, cancellationToken),
```

Comment:
I mean: it's super weird for this static class to be calling another static method in a generic type elsewhere in order to get this functionality.   Either the static should be in DependentTypeFinder (and AbstractReferenceFinder should call that one instead).  Or it should be in some helper class that both of these finders use.